sdk/translation/azure-ai-translation-text/azure/ai/translation/text/_patch.py

```diff
@@ -0,0 +1,67 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+from typing import Union
+from azure.core.pipeline import PipelineRequest
+from azure.core.pipeline.policies import ( SansIOHTTPPolicy, BearerTokenCredentialPolicy, AzureKeyCredentialPolicy )
+from azure.core.credentials import ( TokenCredential, AzureKeyCredential )
+
+from ._client import TextTranslationClient as ServiceClientGenerated
+
+def patch_sdk():
+    """Do not remove from this file.
+
+    `patch_sdk` is a last resort escape hatch that allows you to do customizations
+    you can't accomplish using the techniques described in
+    https://aka.ms/azsdk/python/dpcodegen/python/customize
+    """
+
+class TranslatorCredential:
+    def __init__(self, key: str, region: str) -> None:
```

Comment:
Do we allow update_key?

Reply:
I will resolve this, tracked in previous comment